chain/chain/src/chain.rs

```diff
@@ -810,6 +809,10 @@ impl Chain {
         let prev_epoch_id = self.get_block_header(&head.prev_block_hash)?.epoch_id();
         let epoch_change = prev_epoch_id != &head.epoch_id;
         let mut fork_tail = self.store.fork_tail()?;
+        metrics::TAIL_HEIGHT.set(tail as i64);
+        metrics::FORK_TAIL_HEIGHT.set(fork_tail as i64);
+        metrics::CHUNK_TAIL_HEIGHT.set(self.store.chunk_tail()? as i64);
+        metrics::GC_STOP_HEIGHT.set(gc_stop_height as i64);
```

Comment:
Should this be called at the end of the method since the numbers will be updated?

Reply:
I don't have a strong preference on this. I think either way is fine, since the metric is only pull once every 10 seconds anyways.